Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon.yml

```diff
@@ -371,23 +436,125 @@ tasks:
           transformers:
           - operator: LastArrayElement
     separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1030,
+          "y": 1220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 2cec22a6-b48e-474b-84ee-4aec0f057084
+    type: regular
+    task:
+      id: 2cec22a6-b48e-474b-84ee-4aec0f057084
+      version: -1
+      name: 'Set report number to content '
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      key:
+        simple: report number
+      value:
+        complex:
+          root: Polygon.Analysis
+          accessor: ID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1460,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 1ce58e07-640e-41d0-8e8d-713959491e1c
+    type: condition
+    task:
+      id: 1ce58e07-640e-41d0-8e8d-713959491e1c
+      version: -1
+      name: Is the file type supported?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: match
+          left:
+            value:
+              complex:
+                root: inputs.file_id
```

Comment:
change it to the condition of File.Type + checking if the input.id is in the File.EntryID